packages/dd-trace/src/ci-visibility/log-submission/log-submission-plugin.js

```diff
@@ -0,0 +1,49 @@
+const Plugin = require('../../plugins/plugin')
+const log = require('../../log')
+
+function getWinstonLogSubmissionParameters (config) {
+  const { site, service } = config
+
+  const defaultParameters = {
+    host: `http-intake.logs.${site}`,
+    path: `/api/v2/logs?dd-api-key=${process.env.DD_API_KEY}&ddsource=winston&service=${service}`,
+    ssl: true
+  }
+
+  if (!process.env.DD_AGENTLESS_LOG_SUBMISSION_URL) {
+    return defaultParameters
+  }
+
+  try {
+    const url = new URL(process.env.DD_AGENTLESS_LOG_SUBMISSION_URL)
+    return {
+      host: url.hostname,
+      port: url.port,
+      ssl: url.protocol === 'https:',
+      path: defaultParameters.path
```

Comment:
Again, not sure if this is possible with `winston`, but gzipping the logs would make sense.

Reply:
this complicates the integration considerably and not even the APM-logs integration suggests it, so I'll keep it: https://docs.datadoghq.com/logs/log_collection/nodejs/?tab=winston30#agentless-logging